packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -78,6 +78,7 @@ export function generateRuntimeOptions(
         gcOptions: [undefined, ...gcOptions],
         summaryOptions: [undefined, ...summaryOptions],
         loadSequenceNumberVerification: [undefined],
+        useDataStoreAliasing: [undefined],
```

Comment:
i don't think we create root data stores in stress, might be a good follow up. regardless if you put values here, likely true false, clients will get a mix of the those options values